frontend/src/modules/DatasetsBase/components/DatasetGovernance.js

```diff
@@ -20,6 +22,14 @@ export const DatasetGovernance = (props) => {
       : [{ label: '-', nodeUri: '-' }];
   const tags = dataset.tags.length > 0 ? dataset.tags : ['-'];
 
+  const [ModalOpen, setIsModalOpen] = useState(false);
```

Comment:
nit: Here and all other uses of `ModalOpen` - for consistency the syntax we have for state variables would be `  const [modalOpen, setIsModalOpen] = useState(false);` (lower case `m`)

Reply:
Changed to lowercase in all cases.